config/ec2/config.go

```diff
@@ -194,6 +194,11 @@ func Configure(p *config.Provider) {
 		r.References["source_security_group_id"] = config.Reference{
 			Type: "SecurityGroup",
 		}
+		r.LateInitializer = config.LateInitializer{
+			IgnoredFields: []string{
+				"self", "source_security_group_id",
```

Comment:
[Should we also include](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/security_group_rule#self) `cidr_blocks` and `ipv6_cidr_blocks` together with the `source_security_group_id`? 

I guess they don't get late-initialized since we have already tested the new configuration and hence not included here. Is this correct?